migrations/20250624153928_add_auto_build_to_pr.up.sql

```diff
@@ -0,0 +1,2 @@
+-- Add up migration script here
+ALTER TABLE pull_request ADD COLUMN auto_build_id INTEGER REFERENCES build(id);
```

Comment:
We might as well rename `build_id` to `try_build_id`, to make the distinction explicit (in another migration).